install.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/sh
+set -e
```

Comment:
enabling `pipefail` would allow us to catch errors within pipes, which may otherwise produce garbage output and make debugging more difficult.